integration-tests/cleaner-test/src/main/java/com/cognifide/aet/cleaner/InMemoryDB.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.cleaner;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import de.bwaldvogel.mongo.MongoServer;
+import de.bwaldvogel.mongo.backend.memory.MemoryBackend;
+import java.io.File;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import org.apache.commons.io.FileUtils;
+import org.bson.Document;
+
+public class InMemoryDB {
```

Comment:
Shouldn't all files in this module be in `src/test`?
Why do we need any of the code to be non-test?

Reply:
You're right, I moved all files to `src/test`